packages/form-js-viewer/src/render/components/form-fields/parts/InputAdorner.js

```diff
@@ -0,0 +1,23 @@
+import classNames from 'classnames';
+
+export default function InputAdorner(props) {
```

Comment:
I never heard of `adornment` as this part of a form field element. Maybe just use `InputIcon`?

Reply:
I think it's been long enough and the wording has kinda stuck internally, no? :D

Reply:
Yeah, I think it's okay 👍